irc.py

```diff
@@ -438,11 +439,25 @@ def main() -> None:
                 'authentication, it\'s recommended to only allow local connections\n' \
                 'you can override this with -o')
 
-    sl_client = slack.Slack(args.tokenfile)
+    if "PORT" in environ:
```

Comment:
Can you use single quotes? The rest of the project does.

Reply:
Unless mistakes are made elsewhere too :D

Reply:
Sure thing. I'm like 90% shell programming these days so I avoid single quotes unless I need their variable-avoiding powers.